smt_example_test.go

```diff
@@ -0,0 +1,35 @@
+package smt_test
```

Comment:
Should this not be package `smt`?

Reply:
See https://github.com/pokt-network/smt/pull/33#discussion_r1440698387